.prettierignore

```diff
@@ -0,0 +1 @@
+shared/common-adapters/icon.constants.js
```

Comment:
this doesn't work due to a bug in prettier eslint: https://github.com/prettier/prettier-eslint/issues/67